en/getting_started/README.md

```diff
@@ -4,16 +4,22 @@
 
 This topic explains how to set up a development environment and write a minimal example for all our supported programming languages/platforms. 
 
-## API overview
+## API Overview

```

Comment:
@julianoes This (merged) PR imports API ref "officially" into the doc set. This bit is the main change that needs review - the API overview. Can you please just scan it. You may find it easier to look at in rendered form: https://hamishwillee.gitbooks.io/dronecore/content/en/getting_started/